### PR TITLE
Increase code coverage for older compiler versions

### PR DESCRIPTION
**Pull request by axic**
Part of #180.

Todo:
- [x] test for single file
- [x] test abstract contracts
- [x] test for compiler warnings
- [ ] test for compiler errors
- [x] test translation of gas estimation
- [ ] test with every single release...

### CODE REVIEW

File: test/compiler.js [hunk 1]
```diff
@@ -424,12 +470,56 @@ function runTests (solc, versionText) {
...
+
+      t.test('compiling standard JSON (abstract contract)', function (st) {
+        // <0.1.6 doesn't have this
+        if (!solc.features.multipleInputs) {
+          st.skip('Not supported by solc');
+          st.end();
+          return;
+        }
+
+        var isVersion6 = semver.gte(solc.semver(), '0.5.99');
```
Comment (leonardoalt) [inline test/compiler.js]:
```suggestion
        var isVersion6 = semver.gt(solc.semver(), '0.5.99');
```
?

File: test/compiler.js [hunk 2]
```diff
@@ -453,6 +453,40 @@ function runTests (solc, versionText) {
...
+          'settings': {
+            'outputSelection': {
+              '*': {
+                '*': [ 'evm.bytecode', 'evm.gasEstimates' ]
+              }
+            }
+          },
+          'sources': {
+            'cont.sol': {
+              'content': (isVersion6 ? 'abstract' : '') + 'contract C { function f() public; }'
```
Comment (leonardoalt) [inline test/compiler.js]: Needs a space after `abstract`

Comment (axic) [reply test/compiler.js]:
```suggestion
              'content': (isVersion6 ? 'abstract ' : '') + 'contract C { function f() public; }'
```

**Comment by leonardoalt** [comment]
Maybe we can merge this without the remaining TODOs?

**Pull request merged**